src/postgraphql/schema/procedures/createPgProcedureQueryGqlFieldEntry.ts

```diff
@@ -51,12 +52,17 @@ function createPgSingleProcedureQueryGqlFieldEntry (
     type: fixtures.return.gqlType,
     args: buildObject(argEntries),
 
-    async resolve (_source, args, context): Promise<mixed> {
+    async resolve (_source, args, context, resolveInfo): Promise<mixed> {
       const client = pgClientFromContext(context)
       const input = argEntries.map(([argName], i) => fixtures.args[i].fromGqlInput(args[argName]))
-      const query = sql.compile(sql.query`select to_json(${createPgProcedureSqlCall(fixtures, input)}) as value`)
+      const aliasIdentifier = Symbol()
+      const query = sql.compile(sql.query`
+        select ${getSelectFragment(resolveInfo, aliasIdentifier, fixtures.return.gqlType)} as value,
+        (${sql.identifier(aliasIdentifier)} is null) as is_null
+        from ${createPgProcedureSqlCall(fixtures, input)} as ${sql.identifier(aliasIdentifier)}
+      `)
       const { rows: [row] } = await client.query(query)
-      return row ? fixtures.return.intoGqlOutput(fixtures.return.type.transformPgValueIntoValue(row['value'])) : null
+      return row && !row['is_null'] ? fixtures.return.intoGqlOutput(fixtures.return.type.transformPgValueIntoValue(row['value'])) : null
     },
   }]
 }
```

Comment:
This:
https://github.com/calebmer/postgraphql/blob/333f513eeceb7c989509eee014009938aafd3a46/src/postgraphql/schema/procedures/createPgProcedureQueryGqlFieldEntry.ts#L94

calls into this:
https://github.com/calebmer/postgraphql/blob/333f513eeceb7c989509eee014009938aafd3a46/src/graphql/schema/connection/createConnectionGqlField.ts#L131

which calls into this:
https://github.com/calebmer/postgraphql/blob/333f513eeceb7c989509eee014009938aafd3a46/src/postgres/inventory/paginator/PgPaginatorOrderingOffset.ts#L133-L153

(or this:
https://github.com/calebmer/postgraphql/blob/333f513eeceb7c989509eee014009938aafd3a46/src/postgres/inventory/paginator/PgPaginatorOrderingAttributes.ts#L75-L89
)

which calls this:
https://github.com/calebmer/postgraphql/blob/333f513eeceb7c989509eee014009938aafd3a46/src/postgraphql/schema/procedures/PgProcedurePaginator.ts#L54

and finally:
https://github.com/calebmer/postgraphql/blob/333f513eeceb7c989509eee014009938aafd3a46/src/postgraphql/schema/procedures/createPgProcedureSqlCall.ts#L42


Man this stuff is complex.